tgui/src/interfaces/cryo.ract

```diff
@@ -51,9 +51,9 @@
   <ui-section label='Temperature'>
 		<span class='{{temperatureStatus(adata.cellTemperature)}}'>{{Math.round(adata.cellTemperature)}} K</span>
 	</ui-section>
-	<ui-section label='Door'>
-    <ui-button icon='{{data.isOpen ? "unlock" : "lock"}}' action='door'>{{data.isOpen ? "Open" : "Closed"}}</ui-button>
-	</ui-section>
+  <ui-section label='Door'>
+		<ui-button icon='{{data.isOpen ? "unlock" : "lock"}}' action='door'>{{data.isOpen ? "Open" : "Closed"}}</ui-button>
+    </ui-section>
```

Comment:
step one, section off about 3 hours of time for this task.

step two, install node.js

step three, run install dependency.bat

step four... cryptic error message

step five google error message

step six, hold funeral for the last of the your hair to fall out from stress

step seven: it magically works properly when trying it.

step eight, run compile assets.bat.

step nine: see step four